ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py

```diff
@@ -36,13 +37,14 @@ def get_jobs(yaml_file):
     str_out = out.decode("utf-8")
     return str_out.split("\n")
 
-def get_missed_profiles(e2e_jobs, upgrade_jobs):
+def get_missed_profiles(e2e_jobs, upgrade_jobs, mode='y'):
     """
     Get missed e2e jobs in upgrade jobs
     
     Args:
         e2e_jobs: e2e jobs
         upgrade_jobs: existing upgrade jobs
+        mode: upgrade mode, 'y' or 'z'
```

Comment:
How about use `ystream`, `zstream` to make it more clear ?

Reply:
Not a blocker.  Just let me know if you prefer the current names.

Reply:
ystream, zstream will let us input more characters when run the script...I want it to be easier for users to run scripts.